sdcm/cluster.py

```diff
@@ -1134,21 +1134,33 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         rsa_id_dst_pub = '/tmp/scylla-test-pub'
         mgmt_user = 'scylla-manager'
 
-        if self.is_debian():
+        if self.is_debian8():
             install_transport_https = dedent("""
                 if [ ! -f /etc/apt/sources.list.d/backports.list ]; then sudo echo 'deb http://http.debian.net/debian jessie-backports main' | sudo tee /etc/apt/sources.list.d/backports.list > /dev/null; fi
-                sudo apt-get install apt-transport-https
-                sudo apt-get update
-                sudo apt-get install gnupg-curl
-                sudo apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/Release.key
+                apt-get install apt-transport-https
+                apt-get install gnupg-curl
+                apt-get update
+                apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-jessie/Debian_8.0/Release.key
+                apt-get install -t jessie-backports ca-certificates-java -y
                         """)
             self.remoter.run('sudo bash -cxe "%s"' % install_transport_https)
-            install_open_jdk = dedent("""
-                             sudo apt-get install -t jessie-backports ca-certificates-java -y
-                            sudo apt-get install -y openjdk-8-jre-headless
-                            sudo update-java-alternatives --jre-headless -s java-1.8.0-openjdk-amd64
+
+
+        if self.is_debian9():
```

Comment:
```
if 
elif ...
elif ...
else:
     raise Exception("Unsupported OS")
```

Reply:
in this case added a preceding condition of:
if not (self.is_rhel_like() or self.is_debian() or self.is_ubuntu())